### ISSUE TITLE
[Docs] Remove LookbackSafe and replace with Lookback in the readme

### ISSUE THREAD

**Issue by derekblank** [negativetwelve/react-native-lookback#25]
According to Lookback's docs on [Setting up with Cocoapods](http://success.lookback.io/installing-the-ios-sdk/setting-up/setting-up-with-cocoapods): 

> We previously had both a "HD" and "Safe" version of the SDK. HD is no longer available, and only the appstore-safe version of Lookback is available. It is called just "Lookback", and "LookbackSafe" is no more.

The docs for `react-native-lookback` still reference `LookbackSafe` in the Podfile. I could PR this change if you feel like it doesn't implicate anything seriously.

**Comment by negativetwelve** [comment]
I haven't had a chance to keep up-to-date on Lookback, a PR would be great!

**Comment by peacechen** [comment]
@negativetwelve 
Are you still actively managing this library?  If so, I will submit a PR.

**Comment by negativetwelve** [comment]
@peacechen Yes, I am currently maintaining it but it is not a high priority for me, personally, right now. The project is open for new maintainers if you would like to contribute.
